### PR TITLE
Add 'hoverMode' attribute 

### PR DESCRIPTION
**Pull request by twojtylak**
to enable hover effect on whole row (instead of single cell)

Fixes issue #.

Changes proposed in this pull request:

- 
- 
-

### CODE REVIEW

File: lib/docs.js [hunk 1]
```diff
@@ -38,6 +38,7 @@
  * @param {boolean} [args.persistantSelectionMode=false] - When true, selections will behave as if the command/control key is held down at all times. Conflicts with singleSelectionMode.
  * @param {boolean} [args.singleSelectionMode=false] - When true, selections will ignore the command/control key. Conflicts with persistantSelectionMode.
  * @param {string} [args.selectionMode='cell'] - Can be 'cell', or 'row'.  This setting dictates what will be selected when the user clicks a cell.  The cell, or the entire row respectively.
+ * @param {string} [args.hoverMode='cell'] - Can be 'cell', or 'row'.
```
Comment (ndrsn) [inline lib/docs.js:41]: I would add:

```
This setting dictates whether to highlight either the individual cell, or the entire row when hovering over a cell.
```

**Review by ndrsn — CHANGES_REQUESTED** [review]
Great! Just a minor addition requested in the documentation for `hoverMode`

**Comment by ndrsn** [comment]
Super, thanks for this. I'll release a new version with this later today or tonight, hopefully.

**Comment by twojtylak** [comment]
@ndrsn yeah, thank you

**Comment by ndrsn** [comment]
Released in 0.3.4

**Pull request merged**